ImageFeed/ImagesList/ImagesListViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ViewController.swift
+//  ImageFeed
+//
+//  Created by Илья Чемоданов on 29.04.2023.
+//
+
+import UIKit
+
+private let photosName: [String] = Array(0..<20).map{ "MockImage\($0)" }
```

Comment:
глобальные константы не приняты, строчку лучше поместить во вью-контроллер, так как она связана с ним логически 

к тому же тут можно порпавить форматирование 
```suggestion
private let photosName: [String] = Array(0..<20).map { "MockImage\($0)" }
``` 
фигурные скобки не должны прилипать ни к чему